teps/0074-deprecate-pipelineresources.md

```diff
@@ -160,8 +160,8 @@ that maybe this concept, at least in its current form, isn't required).
     * [TEP-0075 Object/Dictionary support](https://github.com/tektoncd/community/pull/479) must be at a beta level of
       stability
     * [TEP-0076 Array support](https://github.com/tektoncd/community/pull/479) must be at a beta level of stability
-    * [pipeline-to-taskrun experimental custom task](https://github.com/tektoncd/experimental/pull/770) must be used in
-      dogfooding, as a way to prove the viability of a solution to [TEP-0044](https://github.com/tektoncd/community/blob/main/teps/0044-decouple-task-composition-from-scheduling.md)
+    * [pipeline-to-taskrun experimental custom task](https://github.com/tektoncd/experimental/pull/770) must be at an
```

Comment:
So this is gonna be sound naive, but why "pipeline-to-taskrun experimental custom task" is seen as a migration path from PipelineResources ? Wouldn't https://github.com/tektoncd/community/pull/502 or https://github.com/tektoncd/community/blob/main/teps/0044-decouple-task-composition-from-scheduling.md also be considered as alternative ?

Reply:
If I'm not mistake "Pipeline to taskrun" and #502 are both among the alternatives in TEP-0044 - so could refer to TEP-0044 as a more generic requirement.

Reply:
@afrittoli ah yes, I was thinking of TEP-0044 when linking #502.. but couldn't find the PR because it's merged already 😅

Reply:
whoops, jumping to conclusions there! I agree that it makes more sense to block on TEP-0044 👍